src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import Home from "./pages/Home";
+import Video from "./pages/Video";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+
+const router = createBrowserRouter([
```

Comment:
Only thing is like, wat a browser router

Reply:
See comment below about `RouterProvider` and [React Router docs](https://reactrouter.com/en/main/routers/create-browser-router)